challengeFive.js

```diff
@@ -0,0 +1,46 @@
+// First attempt 
+
+let cardNumber = "";
+
+function hideCardNumber(number){
+    const cardNumberString = number.toString();
+    const integers = cardNumberString.split("");
+    
+    for(let i = 0; i < integers.length - 4; i++) {
+        integers[i] = "*";
+        cardNumber += integers[i]
+    }
+   
+    for(let i = integers.length - 4; i < integers.length; i++){
+        cardNumber += integers[i]
+    }
+    console.log(cardNumber)
+}
+
+
+hideCardNumber(123838383848)
+
+
+
+// second attempt 
+
+let encryptedCardNumber = "";
```

Comment:
Same comment as before.

You're not encrypting the card number. You're just hiding the card number. Encrypting the card number would involve using an encryption algorithm and that's cryptography.